config/jobconfig.toml

```diff
@@ -0,0 +1,227 @@
+## This is the configuration file for the job system.
+## This will only be enabled when the config flag LOAD_JOBS_FROM_TXT is enabled.
+## We use a system of keys here that directly correlate to the job, just to ensure they don't desync if we choose to change the name of a job.
+## You are able to change (as of now) four different variables in this file.
+## Total Positions are how many job slots you get in a shift, Spawn Positions are how many you get that load in at spawn. If you set this to -1, it is unrestricted.
+## Playtime Requirements is in minutes, and the job will unlock when a player reaches that amount of time.
+## However, that can be superseded by Required Account Age, which is a time in days that you need to have had an account on the server for.
+## As time goes on, more config options may be added to this file.
+## You can use the admin verb 'Generate Job Configuration' in-game to auto-regenerate this config as a downloadable file without having to manually edit this file if we add more jobs or more things you can edit here.
+## It will always respect prior-existing values in the config, but will appropriately add more fields when they generate.
+## It's strongly advised you create your own version of this file rather than use the one provisioned on the codebase.
+
+## The game will not read any line that is commented out with a '#', as to allow you to defer to codebase defaults.
+## If you want to override the codebase values, add the value and then uncomment that line by removing the # from the job key's name.
+## Ensure that the key is flush, do not introduce any whitespaces when you uncomment a key. For example:
+## "# Total Positions" should always be changed to "Total Positions", no additional spacing. 
+## Best of luck editing!
+
+[AI]
+"# Playtime Requirements" = 180
+"# Required Account Age" = 30
+"# Spawn Positions" = 1
+"# Total Positions" = 1
```

Comment:
Keys with spaces 🤮

Reply:
just realized the comments on these are wrong because theyre inside the string, lol

Reply:
Outstanding move

Reply:
i hate to say it but that was intentional because i couldn't figure out how else I could pass the list into TOML Encode - the whole system is reliant on the keys exactly matching and the "#" forcing a mismatch. i wrote about that here in the [October 3rd post](https://github.com/tgstation/tgstation/pull/70199#discussion_r986316574):

>Okay, I think I understood it now. I'm pretty stupid and lazy (mostly, I couldn't think of a better way to do it), so I just purposefully introduced a key mis-match. Every time we generate jobconfig.toml, we will always write to it with commented out keys.

When I wrote it, the hashtag was meant to act as the pseudo-comment instead of an explicit TOML comment-out, because the end goal was just to get a mismatch between the key in the TOML and any expected keys DM-side. There was probably a better way to do it but it wasn't unintentional